### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by hculea**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,72 @@
...
+[![Version](https://img.shields.io/github/release/1Password/connect-sdk-go.svg)](https://github.com/1Password/connect-sdk-go/releases/)
 
-## Installation
+The 1Password Connect Go SDK provides access to the 1Password Connect API, to facilitate communication with the Connect server hosted on your infrastructure. The library is intended to be used by your applications, pipelines, and other automations to simplify accessing items stored in your 1Password vaults.
 
+## Installation
+To download and install the 1Password Connect Go SDK, as well as its dependencies:
 ```sh
 go get github.com/1Password/connect-sdk-go
 ```
```
Comment (jillianwilson) [inline README.md:13]: Let's also include how to import the library. For example:

Then, import it using:

import (
    "github.com/1Password/connect-sdk-go/connect"
)

File: README.md [hunk 2]
```diff
@@ -1,31 +1,72 @@
...
 
 ## Usage
 
+Below, you can find a selection of the most used functionality of the Connect Go SDK. For more detailed information about the content of the SDK, please refer to the [GoDocs](https://pkg.go.dev/github.com/1Password/connect-sdk-go).
```
Comment (jillianwilson) [inline README.md:17]: Are these go docs up to date?

**Comment by jillianwilson** [comment]
@hculea Hope you don't mind but I changed the order of some of the documentation here. Mind looking over my changes?

File: README.md (hunk 2 above)
Comment (hculea) [reply README.md:59]: I'll address this before the end of the week. :D

Comment (edif2008) [reply README.md:58]: Go docs are updated with the latest version of Readme, as well as the code documentation.
At the moment of writing this comment, the documentation of the structs and functions is up to date (based on the latest release `v1.3.0`). 

The GoDocs readme will be updated when we make a new release with this version of README.

**Review by jillianwilson — APPROVED** [review]

**Pull request merged**